web/src/components/profile/ProfileAddressLockedFields.test.tsx

```diff
@@ -17,17 +23,21 @@ function mockMaterialUI(): typeof materialUi {
 }
 
 describe("ProfileAddressLockedFields", () => {
+  beforeEach(() => {
+    useMockBusiness({});
+  });
```

Comment:
[sand] - I did notice that we're calling useMockBusiness at the beginning of each test so this beforeEach call might be redundant